playgrounds/react-vite/src/tabs/Collectable.tsx

```diff
@@ -0,0 +1,215 @@
+import { Box, Card, Button, Text } from "@0xsequence/design-system";
+import {
+  useMakeOfferModal,
+  useCreateListingModal,
+  useTransferModal,
+  useSellModal,
+  CollectibleCard,
+  useListListingsForCollectible,
+  useListOffersForCollectible,
+  useCurrencies,
+  useBuyModal,
+} from "@0xsequence/marketplace-sdk/react";
+import { useMarketplace } from "../lib/MarketplaceContext";
+import { useAccount } from "wagmi";
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableHead,
+  TableRow,
+  TableCell,
+} from "./../lib/Table/Table";
+import type { Order } from "@0xsequence/marketplace-sdk";
+
+export function Collectible() {
+  const context = useMarketplace();
+
+  return (
+    <Box paddingTop="3" gap="3" flexDirection="column">
+      <Box gap="3">
+        <Box>
+          <CollectibleCard
+            chainId={Number(context.chainId)}
+            collectionAddress={context.collectionAddress}
+            tokenId={context.collectibleId}
+            onCollectibleClick={() => console.log("Collectible clicked")}
+            onOfferClick={() => console.log("Offer clicked")}
+          />
+        </Box>
+        <Card gap="3"></Card>
+      </Box>
+      <Actions />
+      <ListingsTable />
+      <OffersTable />
+    </Box>
+  );
+}
+
+function Actions() {
+  const context = useMarketplace();
+  const { show: openMakeOfferModal } = useMakeOfferModal();
+  const { show: openCreateListingModal } = useCreateListingModal();
+  const { show: openTransferModal } = useTransferModal();
+  const { address } = useAccount();
+
+  if (!address) {
+    return (
+      <Card gap="3">
+        <Text variant="large">Connect Wallet to see collectable actions</Text>
+      </Card>
+    );
+  }
+  return (
+    <Card gap="6" justifyContent="center">
+      <Box gap="3">
+        <Button
+          variant="primary"
+          onClick={() => openMakeOfferModal(context)}
+          label="Make Offer"
+        />
+        <Button
+          variant="primary"
+          onClick={() => openCreateListingModal(context)}
+          label="Buy Item"
+        />
+      </Box>
+      <Box gap="3">
+        <Button
+          variant="primary"
+          onClick={() => openCreateListingModal(context)}
+          label="Create Listing"
+        />
+        <Button
+          variant="primary"
+          onClick={() =>
+            openTransferModal({
+              collectionAddress: context.collectionAddress,
+              chainId: context.chainId,
+              tokenId: context.collectibleId,
+            })
+          }
+          label="Transfer"
+        />
+      </Box>
+    </Card>
+  );
+}
+
+function ListingsTable() {
+  const context = useMarketplace();
+  const { data: listings, isLoading } = useListListingsForCollectible(context);
```

Comment:
I think it's more clear to use the exact params. It also shows how easy the hook to use, wdyt?
```suggestion
  const { data: listings, isLoading } = useListListingsForCollectible({
    chainId: context.chainId,
    collectibleId: context.collectibleId,
    collectionAddress: context.collectionAddress,
  });
```

Reply:
I had this, i didn't like how long the code became. added a compromise, since i think think this is a valid point